source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+   ![Home Assistant cloud language settings](/images/assist/ha-cloud-tts-01.png) 
+1. Go to **Settings** > **Voice assistants** and select **Add assistant**.
+   * Enter a name. You can pick any name that is meaningful to you.
+   ![Enter a name for your voice assistant](/images/assist/assistant-give-name.png)
+   * Select a **Conversation agent**. This is the software that processes your voice stream. 
+   For this tutorial, select **Home Assistant**.
+     * Select the **Language** that you want to speak.
+      ![Select the language you want to speak](/images/assist/assistant-conversation-agent.png)
+   * In the **Speech-to-text** and **Text-to-speech** drop-down menus, select an assistant. For this tutorial, select **Home Assistant Cloud**.
+   * Under **Text-to-speech**, select the **voice** of the assistant. For this tutorial, use the default voice.
+   * Select **Save**.
+   ![Enter a name for your voice assistant](/images/assist/assistant-stt-tts-define.png)
+1. To be able to control your devices over a voice command, you must expose your entities to Assist:
+   * Open the **Expose** tab.
+   ![Expose entities tab](/images/assist/assistant-expose-01.png) 
+   * Select **Expose entities**.
+   * Select all entities you want to be able to control by voice.
+1. Activate your newly created assistant. 
+   * Go to {% my integrations title="**Settings** > **Devices & Services**" %} and in the **Voice over IP** integration, select **Devices**.
+   * Under **Configuration**, in the **Assist pipeline** menu, select your newly created assistant from the list.   
+      ![Activate your newly created assistant](/images/assist/voip-select-pipline.png)
+1. Congratulations! You set up your analog phone to work with Home Assistant. Now pick up the phone and control your device. 
+   * Say a [supported voice command](/docs/assist/builtin_sentences/). For example, *Turn off the light in the kitchen*.
+   * You can also ask a question, such as
+     *  *Is the front door locked?*
+     *  *Which lights are on in the living room?*.
+
+
+## Troubleshooting
+
+Things are not working as expected? This section lists a few steps that may help you troubleshoot issues.
+
+### Troubleshoot Grandstream
+
+If you’re unable to call Home Assistant, confirm the following settings in your Grandstream device’s web interface.
+
+1. On the **FXS Port** tab, check the **Preferred Vocoder** list. 
+   * Make sure that **OPUS** is selected for one of the choices:
+   ![Vocoder OPUS option](/images/assist/grandstream_vocoder.png) 
+1. Under **OPUS Payload type**, make sure the value is `123`. It's the default option.
+   ![Vocoder OPUS payload type](/images/assist/grandstream_opus_payload.png) 
+
+
+### Test if the phrase works without voice
+
+Use this procedure to test if the phrase itself works, without using the voice.
+
+1. Go to the **Developer Tools** and open the **Services** tab.
```

Comment:
This shouldn't be done here. When debugging a pipeline, you can also send in new text/audio.

Reply:
![CleanShot 2023-04-24 at 11 38 39](https://user-images.githubusercontent.com/1444314/234046614-884fcac3-6739-4881-92c9-01356e055e80.png)